rllib/env/__init__.py

```diff
@@ -1,5 +1,3 @@
-from typing import Any
-
 from ray.rllib.env.base_env import BaseEnv
```

Comment:
We should add Unity3DEnv here as well:
from ray.rllib.env.unity3d_env import Unity3DEnv